home/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+# Create your models here.
+
+class Users(models.Model):
+	username = models.CharField(max_length=100)
+	email = models.CharField(max_length=100)
+	first_name = models.CharField(max_length=100)
+	last_name = models.CharField(max_length=100)
+	role = models.CharField(max_length=15)
+
```

Comment:
We probably ought to have a password field too for users who are not logging in with Google. Once we are doing student registrations I don't think we want to require them to have a Google account.